### ISSUE TITLE
TypeError: execv() arg 2 must contain only strings

### ISSUE THREAD

**Issue by simplyianm** [sophron/wifiphisher#15]
```
Traceback (most recent call last):
  File "wifiphisher.py", line 743, in <module>
    dhcp(dhcpconf, ap_iface)
  File "wifiphisher.py", line 407, in dhcp
    Popen(['ifconfig', mon_iface, 'mtu', '1400'], stdout=DN, stderr=DN)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1335, in _execute_child
    raise child_exception
TypeError: execv() arg 2 must contain only strings
```

**Comment by strasharo** [comment]
Instead of opening new tickets, please check the already open ones. This issue is going to be fixed by https://github.com/sophron/wifiphisher/pull/8, however it's still not merged.

**Comment by simplyianm** [comment]
Sorry about that.

**Comment by simplyianm** [comment]
Never mind, the fix does not work for me. Python 2.7.8

Linux ian-lenovo 3.16.0-28-generic #38-Ubuntu SMP Fri Dec 12 17:37:40 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux

**Comment by FurqanHanif** [comment]
#8  is Not Working For me Too ...

**Comment by ParvJain** [comment]
#8 is also not working for (Ubuntu 14.04)

**Comment by ParvJain** [comment]
seems like #16 pull request is fixing this.

**Comment by BLACKZOO** [comment]
I have the same problem. 
I did try all the solutions, but they did not word.

**Comment by BLACKZOO** [comment]
No1 ?

**Comment by BLACKZOO** [comment]
Python 2.7.3
Kali 1.0.9

**Comment by nheafner** [comment]
I to am having the same error

**Comment by italy2010** [comment]
hello
with kali 1.0.9 i get same error,
seem need some fix to run without problem on kali, we should wait.

**Comment by nareshlamgade** [comment]
Ya I am also encountering the same error. Any fix ?

**Comment by ParvJain** [comment]
it just got fixed by #16 for me... but it just jams AP, dosen't creates new one.

**Comment by FurqanHanif** [comment]
#16 not Working For me Too... :/

**Comment by sophron** [comment]
Hi guys,

This has been fixed by #20. Please update to the latest revision. If you are still having issues, please re-open with the exact error message you are getting.

**Comment by mhmurtada** [comment]
#20 dosent work for me. what is the problem?
